javascript/ql/lib/ext/tanstack.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/javascript-all
+      extensible: summaryModel
+    data:
+      - ["@tanstack/angular-query-experimental", "Member[injectQuery]", "Argument[0].ReturnValue.Member[queryFn].ReturnValue", "ReturnValue.Member[data].Awaited", "taint"]
```

Comment:
Could you add a line with `"@tanstack/angular-query"` as well? So when it is removed from experimental status our model will still apply.